src/clients/base-evernode-client.js

```diff
@@ -274,6 +305,43 @@ class BaseEvernodeClient {
      * Note: You need to deserialize HookParameters before passing the transaction to this function.
      * @param {object} tx - The transaction object to be deserialized and extracted.
      * @returns {} The event object in format `{name: string, data: Object}`. Returns `null` if the event is not handled. 
+     * @example
+     *     tx.Memos = TransactionHelper.deserializeMemos(tx?.Memos);
+     *    tx.HookParameters = TransactionHelper.deserializeHookParams(tx?.HookParameters);
+     *    const extracted = await client.extractEvernodeEvent(tx);
+     * 
+     * //example response
+     * {
+     * name: 'HostRegUpdated',
+     * data: {
+     *     transaction: {
```

Comment:
What if we keep the transaction in short form like
```
transaction: {
   host: 'rKrSVLgaKQANTSEv1bY4cT4PVThCzFXpX6',
   Amount: '1',
   ...
}
```

Reply:
changed the transaction comment in short form